frontend/src/metabase-types/api/collection.ts

```diff
@@ -63,7 +63,7 @@ export interface CollectionItem {
   collection_position?: number | null;
   collection_preview?: boolean | null;
   fully_parametrized?: boolean | null;
-  collection?: Collection;
+  collection?: Collection | null;
```

Comment:
This change is required to match the `collection` type to the one in `frontend/src/metabase-types/api/card.ts`